Runtime.ImportModuleError

I've uploaded the .zip file into a new lambda and set S3 as a trigger but getting this error when the lambda is invoked - 

```
Response
{
  "errorType": "Runtime.ImportModuleError",
  "errorMessage": "Error: Cannot find module './utils/index.js'\nRequire stack:\n- /var/task/lib/transformer.js\n- /var/task/index.js\n- /var/runtime/UserFunction.js\n- /var/runtime/index.js",
  "trace": [
    "Runtime.ImportModuleError: Error: Cannot find module './utils/index.js'",
    "Require stack:",
    "- /var/task/lib/transformer.js",
    "- /var/task/index.js",
    "- /var/runtime/UserFunction.js",
    "- /var/runtime/index.js",
    "    at _loadUserApp (/var/runtime/UserFunction.js:100:13)",
    "    at Object.module.exports.load (/var/runtime/UserFunction.js:140:17)",
    "    at Object.<anonymous> (/var/runtime/index.js:43:30)",
    "    at Module._compile (internal/modules/cjs/loader.js:1063:30)",
    "    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)",
    "    at Module.load (internal/modules/cjs/loader.js:928:32)",
    "    at Function.Module._load (internal/modules/cjs/loader.js:769:14)",
    "    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)",
    "    at internal/main/run_main_module.js:17:47"
  ]
}
```

Screenshots of the configurations- 
<img width="950" alt="Screen Shot 2021-05-31 at 4 33 36 PM" src="https://user-images.githubusercontent.com/15661434/120246643-ce4f5a80-c23e-11eb-9dae-a1893edc1e34.png">


<img width="1153" alt="Screen Shot 2021-05-31 at 4 33 26 PM" src="https://user-images.githubusercontent.com/15661434/120246624-c394c580-c23e-11eb-8f61-06aa7c911338.png">

--- a/tools/package.sh
+++ b/tools/package.sh
@@ -1,4 +1,4 @@
 #!/bin/bash
 mkdir -p pkg
 npm ci --prod
-zip pkg/logdna-s3.zip -r node_modules/ index.js package.json lib/*.js
+zip pkg/logdna-s3.zip -r node_modules/ lib/ index.js package.json

